core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -145,6 +149,8 @@ public void start(Boolean isPrimary) throws IOException {
         }
       }
       mAsyncJournalWriter = new AsyncJournalWriter(mJournalWriter);
+      mAsyncAuditLogWriter = new AsyncUserAccessAuditLogWriter();
+      mAsyncAuditLogWriter.start();
```

Comment:
We are starting audit logger on the primary master, for a file system, is there only one primary master?